ceresdb-example/src/test/java/io/ceresdb/ReadmeTest.java

```diff
@@ -51,51 +50,106 @@ public void readmeTest() throws ExecutionException, InterruptedException {
         }
 
         // Create table manually, creating table schema ahead of data ingestion is not required
-        String createTableSql = "CREATE TABLE MY_FIRST_TABLE(" + "ts TIMESTAMP NOT NULL," + "c1 STRING TAG NOT NULL,"
-                                + "c2 STRING TAG NOT NULL," + "c3 DOUBLE NULL," + "c4 STRING NULL," + "c5 INT64 NULL,"
-                                + "c6 FLOAT NULL," + "c7 INT32 NULL," + "c8 INT16 NULL," + "c9 INT8 NULL,"
-                                + "c10 BOOLEAN NULL," + "c11 UINT64 NULL," + "c12 UINT32 NULL," + "c13 UINT16 NULL,"
-                                + "c14 UINT8 NULL," + "c15 TIMESTAMP NULL," + "c16 VARBINARY NULL,"
-                                + "TIMESTAMP KEY(ts)" + ") ENGINE=Analytic";
-        SqlResult result = client.management().executeSql(createTableSql);
+        String createTableSql = "CREATE TABLE IF NOT EXISTS machine_table(" +                                                                                              "ts TIMESTAMP NOT NULL," + //
+                "ts TIMESTAMP NOT NULL," +
+                "city STRING TAG NOT NULL," +
+                "ip STRING TAG NOT NULL," +
+                "cpu DOUBLE NULL," +
+                "mem DOUBLE NULL," +
+                "TIMESTAMP KEY(ts)" + // timestamp column must be specified
+                ") ENGINE=Analytic";
+
+        Result<SqlQueryOk, Err> createResult = client.sqlQuery(new SqlQueryRequest(createTableSql)).get();
```

Comment:
Same as above, `SqlQueryRequest` is needed?

Reply:
`SqlQueryRequest` contains param `tables`, only the java SDK implements automatic parsing of the table name from the Sql statement. This parameter is required for other SDKs, and it is also used for scalability.